charts/testkube-api/values.yaml

```diff
@@ -137,6 +137,9 @@ enableSecretsEndpoint: false
 ## disable running MongoDB migrations on API start
 disableMongoMigrations: false
 
+## install only specified executors with enabled flag
+installOnlySpecifiedExecutors:
```

Comment:
Can we use something like `enabledExecutors: artillery,postman...` in the format of a comma-separated string, where "" means enable all?

Reply:
renamed